src/encoded/static/components/motifs.js

```diff
@@ -85,22 +85,22 @@ export class MotifElement extends React.Component {
 
     render() {
         const element = this.props.element;
-        const targetList = element.targets.map(t => t.label);
+        const targetList = element.targets;
         const targetListLabel = targetList.length > 1 ? 'Targets' : 'Target';
 
         return (
-            <div className="element" id={`element${element.accession}`}>
+            <div className="element" id={`element${element.dataset.split('/')[2]}`}>
```

Comment:
I would probably put `const accession = element.dataset.split('/')[2]` above `return(` and then use that on lines 92 and 95.

Reply:
Sure. Will do.

Reply:
Fixed.